setup.py

```diff
@@ -13,4 +13,7 @@
         [console_scripts]
         colin=colin.cli.colin:cli
     ''',
+    data_files=[("/usr/share/colin/", ["config/default.json",
```

Comment:
Just FYI: this won't work in virtualenv and `pip install --user .`. I'm absolutely fine with that, just sayin'.

Reply:
I vote for .local/share/colin and /usr/local/share/colin

Reply:
So, setting `./share/colin` in `data_files`?

And then try to read in `.local` first?

Reply:
sounds good!